### PR TITLE
Update to version 2.0.1

### PR DESCRIPTION
**Pull request by davidfauth**
Add additional metrics for read replicas

### What does this PR do?

This PR bumps the version to 2.0.1 and added in 3 new metrics to monitor Neo4j Read Replicas.
causal_clustering.read_replica.pull_updates
causal_clustering.read_replica.pull_update_highest_tx_id_requested
causal_clustering.read_replica.pull_update_highest_tx_id_received

### Motivation

Customer has requested additional metrics missing from 2.0.0 version.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: neo4j/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - neo4j
 
+## 2.0.1 / 2022-08-22
+
+* [Added] Added causal_cluster read replica metrics. See [#1509](https://github.com/DataDog/integrations-extras/pull/1509)
```
Comment (maycmlee) [inline neo4j/CHANGELOG.md:5]:
```suggestion
* [Added] Added causal_cluster read replica metrics. See [#1509](https://github.com/DataDog/integrations-extras/pull/1509).
```

File: neo4j/metadata.csv [hunk 2]
```diff
@@ -54,6 +54,9 @@ neo4j.causal_clustering.core.replication_new,gauge,,,,Raft replication new reque
 neo4j.causal_clustering.core.replication_success,gauge,,,,Raft Replication success count.,0,neo4j,causal clustering core replication success,
 neo4j.causal_clustering.core.term,gauge,,,,RAFT Term of this server,0,neo4j,causal clustering core term,
 neo4j.causal_clustering.core.tx_retries,gauge,,,,Transaction retries.,0,neo4j,causal clustering core tx retries,
+neo4j.causal_clustering.read_replica.pull_updates,gauge,,,,The total number of pull requests made by this instance.,0,neo4j,page cache unpins,
+neo4j.causal_clustering.read_replica.pull_update_highest_tx_id_requested,gauge,,,,The highest transaction id requested in a pull update by this instance.,0,neo4j,page cache unpins,
+neo4j.causal_clustering.read_replica.pull_update_highest_tx_id_received,gauge,,,,The highest transaction id that has been pulled in the last pull updates by this instance.,0,neo4j,page cache unpins,
```
Comment (maycmlee) [inline neo4j/metadata.csv:59]:
```suggestion
neo4j.causal_clustering.read_replica.pull_update_highest_tx_id_received,gauge,,,,The highest transaction id that has been pulled in the last pull update by this instance.,0,neo4j,page cache unpins,
```

**Review by maycmlee — COMMENTED** [review]
Small edits for docs!

**Review by maycmlee — APPROVED** [review]
👍 for docs

**Comment by jose-manuel-almaza** [comment]
Could you correct the formatting style errors please?

**Comment by davidfauth** [comment]
> Could you correct the formatting style errors please?

Not sure what the formatting style errors are. Can you give me some guidance on that?

**Review by jose-manuel-almaza — APPROVED** [review]

**Pull request merged**